sql-prepared-plan-cache.md

```diff
@@ -125,12 +125,33 @@ MySQL [test]> select @@last_plan_from_cache;
 
 ## Memory management of Prepared Plan Cache
 
+<CustomContent platform="tidb">
+
+Using Prepared Plan Cache incurs memory overhead. To view the total memory consumption by the cached execution plans of all sessions in each TiDB instance, you can use the [**Plan Cache Memory Usage** monitoring panel](/grafana-tidb-dashboard.md) in Grafana.
+
+> **Note:**
+>
+> Because of the the memory reclaim mechanism of Golang and some uncounted memory structures, the memory displayed in Grafana is not equal to the actual heap memory usage. It is tested that there is a deviation of about 20% between the memory displayed in Grafana and the actual heap memory usage.
```

Comment:
```suggestion
> Because of the the memory reclaim mechanism of Golang and some uncounted memory structures, the memory displayed in Grafana is not equal to the actual heap memory usage. It is tested that there is a deviation of about ±20% between the memory displayed in Grafana and the actual heap memory usage.
```